rootfs/bin/boot

```diff
@@ -48,12 +48,8 @@ function etcd_safe_mkdir {
 	set -e
 }
 
-etcd_set_default protocol "${DEIS_PROTOCOL:-http}"
 etcd_set_default secretKey "${DEIS_SECRET_KEY:-$(openssl rand -base64 64 | tr -d '\n')}"
 etcd_set_default builderKey "${DEIS_BUILDER_KEY:-$(openssl rand -base64 64 | tr -d '\n')}"
-etcd_set_default registrationMode "enabled"
```

Comment:
The controller still supports `registrationMode` options through etcd, so doesn't this setup line need to remain here?